Source/WebKit/UIProcess/mac/WebViewImpl.h

```diff
@@ -751,6 +749,8 @@ ALLOW_DEPRECATED_DECLARATIONS_END
     void nativeMouseEventHandler(NSEvent *);
     void nativeMouseEventHandlerInternal(NSEvent *);
     
+    void fireMouseDidMoveOverElement(NSEvent *);
```

Comment:
Not exactly the right name, since it doesn't call mouseDidMoveOverElement. Maybe `request` or `schedule` or something that indicates it's a later thing?

Alternatively, less of a change from the old name? postFakeMouseDidMoveOverElementCallbackForFlagsChangedEvent? Quite wordy, maybe not that :P

Reply:
To avoid a super wordy name.. I'll go with `scheduleMouseDidMoveOverElement`!

Reply:
https://github.com/WebKit/WebKit/pull/14221/commits/c88025563e2436195514cb0c0a22883be08bf627